src/test/java/expresslibrary/logic/commands/CommandTestUtil.java

```diff
@@ -140,10 +162,9 @@ public static void showBookAtIndex(Model model, Index targetIndex) {
         assertTrue(targetIndex.getZeroBased() < model.getFilteredBookList().size());
 
         Book book = model.getFilteredBookList().get(targetIndex.getZeroBased());
-        final String[] splitName = book.getTitle().title.split("\\s+");
-        model.updateFilteredBookList(new TitleContainsKeywordsPredicate(Arrays.asList(splitName[3])));
+        final String[] splitTitle = book.getTitle().title.split("\\s+");
+        model.updateFilteredBookList(new TitleContainsKeywordsPredicate(Arrays.asList(splitTitle[0])));
```

Comment:
Change this back to 3, it's for `ListBookCommandTest`.
```suggestion
        model.updateFilteredBookList(new TitleContainsKeywordsPredicate(Arrays.asList(splitTitle[3])));
```